statistics_calculations.py

```diff
@@ -117,3 +117,23 @@ def plot_percentage(counter):
     ax.legend(loc='center left', bbox_to_anchor=(1, 0.5))
     plt.xlim(0,100)
     plt.show()
+
+def plot_word(df, word, regex=True):
+    """ Counts the use of a specific word by month, and plot by user
+        Be careful, word might actually be a regex

```

Comment:
Can you elaborate on the "word might be a regex" part?

Reply:
Pretty sure I copied this sentence from the count_word docstring.
And from a quick check, that sentence was in count_word in the original py file you've sent me. I will remove the flag in both functions since I see they're not used. I will also rephrase.

Reply:
its nice to know that I don't understand myself